### PR TITLE
Move tests into an "ops" subdir

### LINKED ISSUE #934: Move tests into an additional `tests/ops/` subdir
### Is your feature request related to a specific problem?

This is purely in preparation for merging the repos

### PR DESCRIPTION
**Pull request by ThomasLaPiana**
# Purpose

- Moves tests into a subdir so that merging repos is easier

# Changes
* [x] move all of the tests into an `ops` subdir
* [x] update any hardcoded references that get broken by the change

# Checklist
- [x] Update [`CHANGELOG.md`](https://github.com/ethyca/fidesops/blob/main/CHANGELOG.md) file
  - [x] Merge in main so the most recent `CHANGELOG.md` file is being appended to
  - [x] Add description within the `Unreleased` section in an appropriate category. Add a new category from the list at the top of the file if the needed one isn't already there.
  - [x] Add a link to this PR at the end of the description with the PR number as the text. example: [#1](https://github.com/ethyca/fidesops/pull/1)
- [ ] Applicable documentation updated (guides, quickstart, postman collections, tutorial, fidesdemo, [database diagram](https://github.com/ethyca/fidesops/blob/main/docs/fidesops/docs/development/update_erd_diagram.md).
- If docs updated (select one):
  - [ ] documentation complete, or draft/outline provided (tag docs-team to complete/review on this branch)
  - [ ] documentation issue created (tag docs-team to complete issue separately)
- [ ] Good unit test/integration test coverage
- [ ] This PR contains a DB migration. If checked, the reviewer should confirm with the author that the [down_revision correctly references the previous migration](https://ethyca.github.io/fidesops/development/contributing_details/#alembic-migrations) before merging
- [ ] The `Run Unsafe PR Checks` label has been applied, and checks have passed, if this PR touches any external services

# Ticket

Fixes #934

### CODE REVIEW

**Comment by ThomasLaPiana** [comment]
reordering the datastore list works 100% of the time 70% of the time!

File: scripts/run_infrastructure.py [hunk 1]
```diff
@@ -10,11 +10,11 @@
 
 DOCKER_WAIT = 5
 DOCKERFILE_DATASTORES = [
-    "mssql",
     "postgres",
     "mysql",
     "mongodb",
     "mariadb",
+    "mssql",
```
Comment (eastandwestwind) [inline scripts/run_infrastructure.py:17]: just curious, did moving `mssql` last change / fix anything?

Comment (ThomasLaPiana) [reply scripts/run_infrastructure.py:17]: in my mind: yes
objectively: hard to say

**Review by eastandwestwind — APPROVED** [review]

**Comment by eastandwestwind** [comment]
@ThomasLaPiana looks good, I'll merge once the conflicts are resolved

**Comment by ThomasLaPiana** [comment]
@eastandwestwind appreciated!

**Pull request merged**